app/models/ingridient.rb

```diff
@@ -1,3 +1,5 @@
 class Ingridient < ApplicationRecord
-  belongs_to :recipe
+  has_many :recipe_ingridients
+  has_many :recipes, through: :recipe_ingridients
+  delegate :unit_amount, to: :recipe_ingridients
```

Comment:
This is wrong and will result in errors. `recipe_ingridients` in this case is a collection and our collection does not respond to `#unit_amount`.